astronomer/providers/sftp/hooks/sftp.py

```diff
@@ -81,6 +96,69 @@ async def _get_conn(self) -> asyncssh.SSHClientConnection:
                 if "private_key" in extra_options:
                     self.private_key = extra_options.get("private_key")
 
+                host_key = extra_options.get("host_key")
+                no_host_key_check = extra_options.get("no_host_key_check")
+
+                if no_host_key_check is not None:
```

Comment:
Should we use `if not no_host_key_check`? I was suggested to write in this way by TP when I sent a PR to OSS airflow

Reply:
I have the same opinion in general, but I guess here the idea is to process the value in case if it set to anything in the extras. I have copied the logic from the SSH hook https://github.com/apache/airflow/blob/88f9c0c41effef13c1014afeef4caa0481c91d9f/airflow/providers/ssh/hooks/ssh.py#L195

Reply:
got it 🙂